pandas/tests/io/pytables/test_append.py

```diff
@@ -822,10 +826,11 @@ def test_append_raise(setup_path):
         df["foo"] = Timestamp("20130101")
         store.append("df", df)
         df["foo"] = "bar"
+        shape = "(30,)" if using_infer_string else "(1, 30)"
```

Comment:
Isn't this still a bug? Not sure why we would expect a different shape with the string data types?

Reply:
I assumed it was due to array-backed vs 1d ndarray backed data. But I haven't checked too deeply.

Reply:
This error message got changed in #60829; we no longer report on the shape.